packages/lerna-changelog-generator/src/index.ts

```diff
@@ -106,8 +112,14 @@ const generateChangelog = async () => {
     console.warn('No new versions have been released');
     return 'false';
   }
+
+  const argv = await yargs(hideBin(process.argv)).argv;
```

Comment:
Либо тут так, чтобы заодно обойтись без assertion'ов ниже 
```suggestion
  const { argv = { projectUrl: '', defaultBranch: '' } } = await yargs(hideBin(process.argv));
```

Reply:
Тут кстати неточность в твоем коде, потому что промис там не yargs(), а yargs().argv
Но мысль я понял, вынес извлечение значений в отдельную функцию и там же есть фолбэки значений.